lib/api.js

```diff
@@ -0,0 +1,281 @@
+var BaseClient = require( "./client" ),
+util = require( "util" );
+
+
+// client
```

Comment:
This comment doesn't provide any value as it simply restates what the function name tells you. Please remove.